### ISSUE TITLE
is it possible to reuse your conditioning?

### ISSUE THREAD

**Issue by batrlatom** [lllyasviel/ControlNet#19]
Hi, I have a question regarding the additional retraining of your model. Imagine that I want to reuse for segmentation conditioning and combine it together with depth estimation conditioning. Is this achievable? 

btw, you should definitely make this project as automatic1111 sd webui plugin ans integrate it to diffusers library. It will be a hit!

**Comment by jacobswan1** [comment]
Indeed! Please consider to adapt codes to Diffusers way for more compatible functions & extensions. Plus SDM v2.1 is indeed better than SDM v1.5.
